### DISCUSSION TITLE
공모전 관련해서 문의 드립니다3

### DISCUSSION THREAD

**Discussion by ghost** [naverz/zepeto-studio-kor#960] [5. Q&A]
제페토 월드 공모전에 참가중인데 궁금한 점이 있어서 문의 드립니다.

현재 저희 월드가 서비스 되고 있는데, 저희를 포함해서 <대학생 공모전 월드>라고 표시된 월드가 2개입니다.
혹시 지금 공모전에 출품된 월드들이 정식으로 서비스 되고 있는게 맞나요?🤔

그리고, 혹시 공모전에서 순위에 들면 마케팅에 도움이 될만한 어떤 혜택이 있나요?
만약 예정되어 있지 않다면 저희가 직접 마케팅을 하려고 합니다!👍

알려주세요🙏

**Comment by Zepeto-TechEvangelist** [comment]
안녕하세요~ 공모전에 참가하여 주셔서 감사합니다!
심사를 거쳐 문제가 없다고 판단된 월드는 모두 정식으로 출시가 됩니다 :)

순위에 따른 혜택은 말씀하신 것처럼 마케팅 부분을 고려하고 있습니다만
자세한 것은 7월 중에 공개될 공지 사항을 참고하여 주세요.
감사합니다.
